### ISSUE TITLE
blocking after sending message

### ISSUE THREAD

**Issue by dani230** [wireapp/wire-android#1950]
## The problem
After sending a message, the app is completely blocked for one to three seconds, does not allow you to scroll, write a message or change chat.

## Environment

* Wire version that exhibits the issue:3.25.685 and 3.24.684

* Android OS version used to run Wire: 8.1.0
* Is this a custom firmware or a stock one: Stock
* Mobile phone model/manufacturer: Asus zenfone 4 max
* Mobile network type (EDGE/LTE/Wi-Fi/Offline):Wi-FI

## Details
Write and send any message, the blocking time is variable.

**Comment by marcoconti83** [comment]
Hi @dani230 ,


Thanks for opening the issue, do you still have this issue with 3.26?

**Comment by dani230** [comment]
Yes on 3.26.694, it also takes a long time to download new messages.

**Comment by dani230** [comment]
I noticed something, this happens even in version 3.27.697, and only happens with the chat with a friend who has a history of a year, but not with another that the history is quite minor.

**Comment by marcoconti83** [comment]
@dani230 thanks for reporting the additional information, it might be useful.

Just to set the expectation, while this is a pretty annoying bug, we are currently focusing on fixing other bugs from which the user can not easily recover. We will get to this one at a later stage. If anyone in the community feels like investigating this one before we have a chance to look at it ourselves, any profiling done by someone that has a similar issue would definitely help figure out where the problem is.

**Comment by csritu** [comment]
I have some further details on this.

First, in private chats with fewer messages in the history, it doesn't block. It's not universal after the database as a whole has grown to a certain size, but rather bound to the history of the specific chat.

Second, the effect is much reduced in group chats even with much larger histories than the histories of private chats. With only 3 people, the effect is still there but reduced. With four people, it's possibly nonexistent, but that may change if the history grows much, much more. I couldn't say.

My final insight is that there's no blocking when I reply from the notification. Based on this one, I suspect this bug can be mitigated entirely rather than just an inevitable product of a growing chat history.
